### DISCUSSION TITLE
Deprecated set-output warning in workflow without set-output

### DISCUSSION THREAD

**Discussion by Ghostbird** [community/community#36767] [Actions] [answered]
I keep getting these warnings on my build:

![image](https://user-images.githubusercontent.com/1202149/196918434-2719c762-abb1-4666-a433-fab50afa54c1.png)

I've triple checked, and my workflow _never_ references set-output anywhere.

How can I fix this?

**Comment by antonblr** [comment, marked as answer]
Hi @Ghostbird. It is hard to tell without seeing and actual workflow yaml, but my guess will be that this is coming from some of the actions you're using ([actions/checkout](https://github.com/actions/checkout/issues/959) had this and so on).

**Reply by Ghostbird** [reply]
Yeah, after analysis of the frequency in various steps of the pipeline, on last nights run, I just found that it was because of https://github.com/actions/download-artifact
A fixed version had been released this morning and today my runs no longer show the warning.
It would be nice if Github changed the warning a bit, so it becomes clear whether it's generated from my own commands, or from an action I use. However since the underlying technical reason is identical, I'm not sure whether that's easily feasible.

**Comment by ossanna16** [comment]
Thank you for asking a good question @Ghostbird, I'm glad you found our community 🙂
